### PR TITLE
simplify video timestamping - remove video time mode 

### PR DESCRIPTION
**Pull request by Wumpf**
### What

* fixes https://github.com/rerun-io/rerun/issues/7472

### Checklist
* [x] I have read and agree to [Contributor Guide](https://github.com/rerun-io/rerun/blob/main/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/rerun-io/rerun/blob/main/CODE_OF_CONDUCT.md)
* [x] I've included a screenshot or gif (if applicable)
* [x] I have tested the web demo (if applicable):
  * Using examples from latest `main` build: [rerun.io/viewer](https://rerun.io/viewer/pr/7476?manifest_url=https://app.rerun.io/version/main/examples_manifest.json)
  * Using full set of examples from `nightly` build: [rerun.io/viewer](https://rerun.io/viewer/pr/7476?manifest_url=https://app.rerun.io/version/nightly/examples_manifest.json)
* [x] The PR title and labels are set such as to maximize their usefulness for the next release's CHANGELOG
* [x] If applicable, add a new check to the [release checklist](https://github.com/rerun-io/rerun/blob/main/tests/python/release_checklist)!
* [x] If have noted any breaking changes to the log API in `CHANGELOG.md` and the migration guide

- [PR Build Summary](https://build.rerun.io/pr/7476)
- [Recent benchmark results](https://build.rerun.io/graphs/crates.html)
- [Wasm size tracking](https://build.rerun.io/graphs/sizes.html)

To run all checks from `main`, comment on the PR with `@rerun-bot full-check`.

### CODE REVIEW

File: crates/store/re_types/src/components/video_timestamp_ext.rs [hunk 1]
```diff
@@ -4,18 +4,18 @@ impl VideoTimestamp {
...
     pub fn from_seconds(seconds: f64) -> Self {
-        crate::datatypes::VideoTimestamp::from_nanoseconds((seconds * 1e9) as i64).into()
+        crate::datatypes::VideoTimestamp::from_nanos((seconds * 1e9) as i64).into()
     }
 
     /// Create new timestamp from milliseconds since video start.
     #[inline]
     pub fn from_milliseconds(milliseconds: f64) -> Self {
-        crate::datatypes::VideoTimestamp::from_nanoseconds((milliseconds * 1e6) as i64).into()
+        crate::datatypes::VideoTimestamp::from_nanos((milliseconds * 1e6) as i64).into()
```
Comment (emilk) [inline crates/store/re_types/src/components/video_timestamp_ext.rs:13]:
```suggestion
        crate::datatypes::VideoTimestamp::from_nanos((milliseconds * 1e6).round() as i64).into()
```

File: crates/store/re_types/src/components/video_timestamp_ext.rs [hunk 2]
```diff
@@ -4,18 +4,18 @@ impl VideoTimestamp {
     /// Create new timestamp from seconds since video start.
     #[inline]
     pub fn from_seconds(seconds: f64) -> Self {
-        crate::datatypes::VideoTimestamp::from_nanoseconds((seconds * 1e9) as i64).into()
+        crate::datatypes::VideoTimestamp::from_nanos((seconds * 1e9) as i64).into()
```
Comment (emilk) [inline crates/store/re_types/src/components/video_timestamp_ext.rs:7]:
```suggestion
        crate::datatypes::VideoTimestamp::from_nanos((seconds * 1e9).round() as i64).into()
```

File: crates/store/re_types/definitions/rerun/components/video_timestamp.fbs [hunk 3]
```diff
@@ -4,9 +4,10 @@ namespace rerun.components;
 /// Timestamp inside a [archetypes.AssetVideo].
 struct VideoTimestamp (
   "attr.docs.unreleased",
+  "attr.rerun.experimental",
+  "attr.arrow.transparent",
   "attr.rust.derive": "Copy, PartialEq, Eq, Default",
-  "attr.rust.repr": "transparent",
-  "attr.rerun.experimental"
+  "attr.rust.repr": "transparent"
```
Comment (emilk) [inline crates/store/re_types/definitions/rerun/components/video_timestamp.fbs:10]: You didn't like the lexicographical sorting? 😢

**Review by emilk — APPROVED** [review]

File: crates/store/re_types/definitions/rerun/components/video_timestamp.fbs (hunk 3 above)
Comment (Wumpf) [reply crates/store/re_types/definitions/rerun/components/video_timestamp.fbs:10]: hear me out 😄 : By putting experimental & docs at the top, they can be removed without messing with the non-trailing comma at the end!

everything else I don't have an excuse ;-)

**Pull request merged**